apps/api/src/domain/models/digital-text/digital-text.from-event-history.spec.ts

```diff
@@ -35,6 +47,52 @@ const digitalTextCreated = new DigitalTextCreated(
     dummySystemUserId
 );
 
+const dummyPageIdentifier = '21';
+
+const addPageForDigitalText = clonePlainObjectWithOverrides(
+    testFsaMap.get(ADD_PAGE_FOR_DIGITAL_TEXT) as CommandFSA<AddPageForDigitalText>,
+    {
+        payload: {
+            aggregateCompositeIdentifier: { id },
+            identifier: dummyPageIdentifier,
+        },
+    }
+);
+
+const pageAddedForDigitalText = new PageAddedForDigitalText(
+    addPageForDigitalText.payload,
+    buildDummyUuid(155),
+    dummySystemUserId
+);
+
+const idForUserWithAccessToDigitalText = buildDummyUuid(45);
+
+/**
+ * TODO: note that the aggregate composite identifier should be for the resource
+ * and not the user.
+ *
+ * We need to fix the grantResourceReadAccessToUser command
```

Comment:
Do this now. It is very important.

Reply:
[This](https://www.pivotaltracker.com/story/show/186326559) will be the immediately next story. We want to keep this fix on a separate, atomic, commit.